### DISCUSSION TITLE
Data Warehouse integration

### DISCUSSION THREAD

**Discussion by aaron-larue** [twentyhq/twenty#6088] [Ideas]
Would love to be able to sync data to a data warehouse (or to an ETL tool like Fivetran) so we can run reporting off of CRM data.

**Comment by BOHEUS** [comment]
@aaron-larue core team is currently working on extensibility, once SDK is stable and v1.0 is released (most likely this quarter), you can write your own integration or you can use existing workflows to write such integration natively
